### PR TITLE
Added few alternatives to core_concepts

### PR DESCRIPTION
**Pull request by DanyC97**
Also pulled in https://github.com/dgkanatsios/CKAD-exercises/pull/15 - credit to @abdennour

### CODE REVIEW

File: a.core_concepts.md [hunk 1]
```diff
@@ -130,7 +136,7 @@ kubectl create namespace myns -o yaml --dry-run
 <p>
 
 ```bash
-kubectl create resourcequota myrq -o yaml --dry-run
+kubectl create quota myrq --hard=cpu=1,memory=1G,pods=2,services=3,secrets=5,persistentvolumeclaims=10 --dry-run -o yaml
```
Comment (dgkanatsios) [inline a.core_concepts.md]: Good point, I'd write that as 
```bash
kubectl create quota myrq --hard=cpu=1,memory=1G,pods=2 --dry-run -o yaml
```
I think it's simpler :)

File: a.core_concepts.md [hunk 2]
```diff
@@ -189,6 +200,18 @@ wget -O- 10.1.1.131:80
 exit
 ```
 
+Alternatively you can also try a more advanced option:
+
+```bash
+# Get IP of the nginx pod
+NGINX_IP = $(kubectl get pod nginx -o jsonpath='{.status.podIP}');
```
Comment (dgkanatsios) [inline a.core_concepts.md]: I think the correct way is
```bash
NGINX_IP=$(kubectl get pod nginx -o jsonpath='{.status.podIP}')
```

**Comment by dgkanatsios** [comment]
Just a few changes, otherwise LGTM. Thanks a lot for the contribution!

File: a.core_concepts.md (hunk 2 above)
Comment (DanyC97) [reply a.core_concepts.md]: doh ,my bad.

File: a.core_concepts.md (hunk 1 above)
Comment (DanyC97) [reply a.core_concepts.md]: agreed

**Comment by DanyC97** [comment]
@dgkanatsios can you please review it again ?

File: a.core_concepts.md [hunk 3]
```diff
@@ -189,6 +200,18 @@ wget -O- 10.1.1.131:80
 exit
 ```
 
+Alternatively you can also try a more advanced option:
+
+```bash
+# Get IP of the nginx pod
+NGINX_IP=$(kubectl get pod nginx -o jsonpath='{.status.podIP}');
```
Comment (dgkanatsios) [inline a.core_concepts.md]: doesn't work for me if you include the colon (`;`) at the end. Works if you take it out, give it a shot!

**Comment by dgkanatsios** [comment]
remove the `;` and we're good to go!

File: a.core_concepts.md (hunk 3 above)
Comment (DanyC97) [reply a.core_concepts.md]: sorry !, done

**Comment by dgkanatsios** [comment]
LGTM, thanks for the contribution!

**Pull request merged**